src/selinux/swtpm_svirt.te

```diff
@@ -25,9 +25,11 @@ allow svirt_t virt_var_run_t:file { create getattr open read unlink write };
 allow svirt_t virt_var_run_t:sock_file { create setattr };
 
 allow svirt_tcg_t virtd_t:fifo_file { write read };
-allow svirt_tcg_t virt_var_run_t:sock_file { create setattr };
+allow svirt_tcg_t virt_var_run_t:sock_file { create setattr unlink };
 allow svirt_tcg_t virt_var_run_t:file { create getattr open read unlink write };
 allow svirt_tcg_t virt_var_run_t:dir { write add_name remove_name };
 allow svirt_tcg_t swtpm_exec_t:file { entrypoint map };
+allow svirt_tcg_t user_tmp_t:sock_file { create setattr unlink };
+allow svirt_tcg_t self:process ptrace;
```

Comment:
@elmarco Do you have any idea why this may be necessary? It corresponds to this:

```
type=AVC msg=audit(1691788638.871:1062): avc:  denied  { ptrace } for  pid=81390 comm="swtpm" scontext=unconfined_u:unconfined_r:svirt_tcg_t:s0:c8,c265 tcontext=unconfined_u:unconfined_r:svirt_tcg_t:s0:c8,c265 tclass=process permissive=1
```

I don't see an explicit ptrace() in the swtpm code.

Reply:
that ptrace() is suspicious.. check where it comes from with:

`bpftrace -e 'tracepoint:syscalls:sys_enter_ptrace { print(pid); print(ustack()); }'` ?

(gdb --pid .. to find the stack symbols)

Reply:
Today I could not recreate the issue anymore I remove the ptrace rule now.